packages/grafana-ui/src/components/Button/ButtonContent.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import cx from 'classnames';
+
+type Props = {
+  icon?: string;
+  className: string;
+  iconClassName: string;
+  children: React.ReactNode;
+};
+export function ButtonContent(props: Props) {
+  const { icon, className, iconClassName, children } = props;
+  return icon ? (
+    <span className={className}>
+      <i className={cx([icon, iconClassName])} />
+      <span>{children}</span>
+    </span>
+  ) : (
+    <>children</>
```

Comment:
```suggestion
    <>{children}</>
```

Reply:
huh nice catch, not sure how I missed that one :D